site/docs/features/management.md

```diff
@@ -5,34 +5,107 @@ service can be scheduled and Nessie reports the outcome of each scheduled operat
 
-!!! info
-    Table management services are currently in progress and are not yet included in a released version of Nessie.
-
 ## Garbage Collection
 
-Since Nessie is maintaining many different versions of metadata and data-pointers simultaneously,
+Since Nessie is maintaining many versions of metadata and data-pointers simultaneously,
 you must rely on Nessie to clean up old data. Nessie calls this garbage collection.
 
-Garbage collection policies are configurable and can be either destructive or instructive.
-To configure garbage collection, you must set the following settings:
-
-### Collection Types
+There are at least two steps to a garbage collection action. The first steps are instructive, and the last step is destructive.
 
-|Collection|Modes|Notes|
-|-|-|-|
-|Unreferenced Data|Instructive or Destructive|Deletes any objects that haven't been **directly** referenced by a tag or branch for 30 days.|
-|Unreferenced Commit|Deletes internal Nessie objects that are no longer referenced|Destructive|
-|Unreferenced Metadata|Deletes metadata objects that are no longer referenced[^1]|
-|Referenced Commit & Metadata|Deletes historical commit information that is in a valid commit tree but older than a certain date|
-
-!!! danger
-    Because of the nature of data sizes, Nessie treats any historical commit as "unreferenced"
-    if it is not **directly** referenced by a tag or branch. **This is very different than traditional Git**.
-    If you want to also maintain historical versions of data beyond your garbage collection time,
-    you should leverage Time-based AutoTagging (below).
+!!! info
+    currently the GC algorithm only works for  
+
+### Identify Unreferenced Assets
+
+This is a spark job which should be run periodically to identify no longer referenced assets. Assets are defined as the set of
+files, records, entries etc that make up a table, view or other Nessie object. For example, iceberg assets are:
+ * manifest files
+ * manifest lists
+ * data files
+ * metadata files
+ * the entire table directory on disk (if it is empty)
+
+To be marked as unreferenced an asset must either:
+ 1. No longer referenced by any branch or tag. For example, an entire branch was deleted, and a table on that branch is no longer accessible.
+ 2. Assets created in a commit which has passed the (configurable) commit age. *If they are not referenced by newer commits*
+
+Identifying unreferenced assets is a non-destructive action. The result of the spark job is a Spark DataFrame of all the
+unreferenced assets. This dataframe is stored in an iceberg table managed by nessie at a configurable key. This table is
+referencable in Nessie so can be examined via Spark or any other Nessie/Iceberg compatible engine. An example of the table output
+is shown below.
+
+![Screenshot](../img/gc-table.png)
+
+This action is designed to run concurrently to other workloads and can/should be run regularly. This table is used as input
```

Comment:
maybe s/`concurrently to other workloads`/`concurrently to normal operational workloads`/ ?

Reply:
fixed